ReportingServicesTools/Functions/Admin/Set-RsDatabase.ps1

```diff
@@ -68,42 +92,55 @@ function Set-RsDatabase
 
         [switch]
         $IsRemoteDatabaseServer,
-        
+
         [Parameter(Mandatory = $True)]
         [Alias('DatabaseName')]
         [string]
         $Name,
 
         [switch]
         $IsExistingDatabase,
-        
+
+        [switch]
+        $IsExistingDatabaseUser,
+
+        [switch]
+        $HasDatabaseUserRights,
+
         [Parameter(Mandatory = $true)]
         [Alias('Authentication')]
         [Microsoft.ReportingServicesTools.SqlServerAuthenticationType]
         $DatabaseCredentialType,
-        
+
         [System.Management.Automation.PSCredential]
         $DatabaseCredential,
-        
+
+        [Parameter]
+        [Microsoft.ReportingServicesTools.SqlServerAuthenticationType]
+        $AdminDatabaseCredentialType,
```

Comment:
Should have default of current user (windows) otherwise existing scripts will fail (as the integration test failed)

Reply:
I thought I got that, but fixing now.